dashboard/src/components/AppView/ServicesTable/ServicesTable.test.tsx

```diff
@@ -34,51 +33,55 @@ it("renders a message if there are no services or ingresses", () => {
   ).toContain("The current application does not contain any service");
 });
 
-it("renders a table with a service with a LoadBalancer", () => {
-  const service = {
-    metadata: {
-      name: "foo",
-    },
-    spec: {
-      type: "LoadBalancer",
-    },
-  } as IResource;
-  const services = [{ isFetching: false, item: service }];
-  const wrapper = shallow(<ServiceTable services={services} />);
-  expect(wrapper.find(ServiceItem).props()).toMatchObject({
-    service: { metadata: { name: "foo" }, spec: { type: "LoadBalancer" } },
-  });
-});
+// these tests are no longer relevant as ServicesTable doesn't have the Service
+// resources anymore
+// TODO: update or remove
 
-it("renders a table with a service with two services", () => {
-  const service1 = {
-    metadata: {
-      name: "foo",
-    },
-    spec: {
-      type: "LoadBalancer",
-    },
-  } as IResource;
-  const service2 = {
-    metadata: {
-      name: "bar",
-    },
-    spec: {
-      type: "ClusterIP",
-    },
-  } as IResource;
-  const services = [{ isFetching: false, item: service1 }, { isFetching: false, item: service2 }];
-  const wrapper = shallow(<ServiceTable services={services} />);
-  expect(
-    wrapper
-      .find(ServiceItem)
-      .at(0)
-      .props(),
-  ).toMatchObject({ service: service1 });
-  expect(
-    wrapper
-      .find(ServiceItem)
-      .at(1)
-      .props(),
-  ).toMatchObject({ service: service2 });
-});
+// it("renders a table with a service with a LoadBalancer", () => {
```

Comment:
An alternative to commenting out the code would be to skip it (`xit` instead of `it`). At least you will remember to implement it.

Reply:
do you plan to remove the tests?

Reply:
good point

Reply:
but if the tests doesn't apply anymore, shoudln't we delete them? (as far as they are applied where they belong). I am not sure if I am missing something

Reply:
Yes, I just commented them out temporarily because it wasn't compiling in the dev environment